### DISCUSSION TITLE
I made a colab for Derfuu/comfyui-derfuu-math-and-modded-nodes 🥳

### DISCUSSION THREAD

**Discussion by camenduru** [Comfy-Org/ComfyUI#252] [Show and tell]
ComfyUI Derfuu Math and Modded Nodes 
https://civitai.com/models/21558/comfyui-derfuu-math-and-modded-nodes

Colab (derfuu_comfyui_colab)
https://github.com/camenduru/comfyui-colab

**Comment by WASasquatch** [comment]
I wonder if this is something that could be added to ComfyUI to launch from anywhere.

Basically a config where you can give it either a github raw address to a single `.py` node, or a github repo to download from the `custom_nodes` folder (thus installing the node as a folder within custom nodes and relying on repos `__init__.py`)
